dbt/include/databricks/macros/materializations/incremental/incremental.sql

```diff
@@ -78,6 +78,7 @@
       Also, why does not either drop_relation or adapter.drop_relation work here?!
       --#}
     {%- endif -%}
+    {% do apply_liquid_clustered_cols(target_relation) %}
```

Comment:
only call this logic on an is_incremental run, otherwise liquid clustering is already specified in the create statement.